src/plugins/notification_notify2.py

```diff
@@ -1,15 +1,25 @@
 # -*- coding: utf-8 -*-
+"""
+src/plugins/notification_notify2.py
+===================================
+"""
+# pylint: disable=import-error
 
 import gi
-gi.require_version('Notify', '0.7')
 from gi.repository import Notify
 
+gi.require_version('Notify', '0.7')
+
```

Comment:
This not works in such way. `gi.require_version()` should be called before import.